docs/security.md

```diff
@@ -26,11 +26,14 @@ Learn more in our [pings documentation](https://docs.sourcegraph.com/admin/pings
 
 [Sourcegraph can be configured to enforce repository permissions from code hosts.](https://docs.sourcegraph.com/admin/repo/permissions). Unit and integration tests protect the correctness of these permissions checks.
 
-## Customer data
+## Data access
 
+When you run Sourcegraph on your own infrastructure, all application and user access logs are stored locally only. Sourcegraph employees and contractors never have access to your Sourcegraph or the code, user data, or application data stored in it (excluding any manual intervention, such as e-mailing logs for support purposes).
```

Comment:
When running Sourcegraph on your own infrastructure, all application logs are stored locally, and never shared with Sourcegraph. Sourcegraph employees and contractors never have access to your Sourcegraph instance, or any of its data, unless explicitly shared for troubleshooting purposes.